### DISCUSSION TITLE
Модуль обратного звонка (callback)

### DISCUSSION THREAD

**Discussion by cblvnche** [mikopbx/Core#323] [Q&A] [answered]
Не могу найти модуль обратного звонка (callback)

**Comment by boffart** [comment, marked as answer]
Есть вот такой [модуль](https://github.com/mikopbx/ModuleQueueCallBack) 
https://github.com/mikopbx/ModuleQueueCallBack

Вот тут инструкции по нему
https://wiki.mikopbx.ru/callback-queues

Модуль в разработке, релизов не было.
